test/libyul/yulOptimizerTests/blockOutliner/only_assign.yul

```diff
@@ -0,0 +1,30 @@
+{
+    let a
+    let b
+    let c
+    {
+        a := mul(b,c)
+    }
+    {
+        b := mul(c,a)
+    }
+    {
+        c := mul(a,b)
+    }
+}
+// ====
+// step: blockOutliner
+// ----
+// {
+//     let a
+//     let b
+//     let c
+//     { a := _1(c, b) }
```

Comment:
Why are the arguments reversed?

Reply:
That's the visitation order in ``ASTWalker::operator()(FunctionalInstruction const&)``.

Reply:
It's "reversed twice", though, i.e. still correct, so I don't see any harm and I'd say the order of occurrence in AST visitation is a natural order for the arguments.

Reply:
If it's not a big deal, I would still suggest to reverse the variables, because it might be easier to read in the simplest cases.